templates/quiz/results-attempt-questions-list.php

```diff
@@ -16,92 +17,96 @@
-					<p class="llms-quiz-results-label student-answer"><?php _e( 'Selected answer: ', 'lifterlms' ); ?></p>
-					<?php echo $attempt_question->get_answer(); ?>
-				</div>
-			<?php endif; ?>
+				<?php if ( ! $attempt_question->is_correct() ) : ?>
+					<?php if ( llms_parse_bool( $quiz_question->get_quiz()->get( 'show_correct_answer' ) ) ) : ?>
+						<?php if ( in_array( $quiz_question->get_auto_grade_type(), array( 'choices', 'conditional' ) ) ) : ?>
+							<div class="llms-quiz-attempt-answer-section llms-correct-answer">
+								<p class="llms-quiz-results-label correct-answer"><?php _e( 'Correct answer: ', 'lifterlms' ); ?></p>
+								<?php echo $attempt_question->get_correct_answer(); ?>
+							</div>
+						<?php endif; ?>
+					<?php endif; ?>
 
-			<?php if ( ! $attempt_question->is_correct() ) : ?>
-				<?php if ( llms_parse_bool( $quiz_question->get_quiz()->get( 'show_correct_answer' ) ) ) : ?>
-					<?php if ( in_array( $quiz_question->get_auto_grade_type(), array( 'choices', 'conditional' ) ) ) : ?>
-						<div class="llms-quiz-attempt-answer-section llms-correct-answer">
-							<p class="llms-quiz-results-label correct-answer"><?php _e( 'Correct answer: ', 'lifterlms' ); ?></p>
-							<?php echo $attempt_question->get_correct_answer(); ?>
+					<?php if ( llms_parse_bool( $quiz_question->get( 'clarifications_enabled' ) ) ) : ?>
+						<div class="llms-quiz-attempt-answer-section llms-clarifications">
+							<p class="llms-quiz-results-label clarification"><?php _e( 'Clarification: ', 'lifterlms' ); ?></p>
+							<?php echo $quiz_question->get( 'clarifications' ); ?>
 						</div>
 					<?php endif; ?>
 				<?php endif; ?>
 
-				<?php if ( llms_parse_bool( $quiz_question->get( 'clarifications_enabled' ) ) ) : ?>
-					<div class="llms-quiz-attempt-answer-section llms-clarifications">
-						<p class="llms-quiz-results-label clarification"><?php _e( 'Clarification: ', 'lifterlms' ); ?></p>
-						<?php echo $quiz_question->get( 'clarifications' ); ?>
+
+				<?php if ( $attempt_question->has_remarks() ) : ?>
+					<div class="llms-quiz-attempt-answer-section llms-remarks">
+						<p class="llms-quiz-results-label remarks"><?php _e( 'Instructor remarks: ', 'lifterlms' ); ?></p>
+						<div class="llms-remarks"><?php echo wpautop( $attempt_question->get( 'remarks' ) ); ?></div>
 					</div>
 				<?php endif; ?>
-			<?php endif; ?>
-
 
-			<?php if ( $attempt_question->has_remarks() ) : ?>
-				<div class="llms-quiz-attempt-answer-section llms-remarks">
-					<p class="llms-quiz-results-label remarks"><?php _e( 'Instructor remarks: ', 'lifterlms' ); ?></p>
-					<div class="llms-remarks"><?php echo wpautop( $attempt_question->get( 'remarks' ) ); ?></div>
-				</div>
-			<?php endif; ?>
+			</section>
 
-		</section>
-
-	</li>
+		</li>
 <?php endforeach; ?>
+<?php } else { ?>
+	<p><?php esc_html_e( 'The quiz is still ongoing. You can resume your attempt by clicking the \'Resume Quiz\' button.', 'lifterlms' ); ?></p>
```

Comment:
```suggestion
	<p><?php esc_html_e( "The quiz is still ongoing. You can resume your attempt by clicking the 'Resume Quiz' button.", 'lifterlms' ); ?></p>
```
let's not complicate translators job :D